x-pack/plugins/lens/public/types.ts

```diff
@@ -56,7 +61,7 @@ export interface Datasource<T = unknown> {
  */
 export interface DatasourcePublicAPI {
   getTableSpec: () => TableSpec;
-  getOperationForColumnId: (columnId: string) => Operation;
+  getOperationForColumnId: (columnId: string) => Operation | null;
```

Comment:
I think that undefined tends to be an easier case to handle / generate than null. `Operation | undefined` would be my preference here.

Reply:
I'd like to discuss that idea later- `null` seems most appropriate here: `The value null represents the intentional absence of any object value.`